actors/evm/src/lib.rs

```diff
@@ -144,14 +145,15 @@ impl EvmContractActor {
         method: u64,
         input_data: &[u8],
         readonly: bool,
+        delegate: Option<Cid>,
```

Comment:
Wdyt about renaming `delegate` to something very explicit like `delegateCallBytecode`. I'm afraid that "delegate" alone can be confusing since this word is also used for delegated addresses (`Payload::Delegate`).

Reply:
not really in this context.

Reply:
I think this is still pretty valid. We should probably just call this `with_code`.